File: apps/py/example_UnaryToStream.py
import time
from core.lib.decorator import channel


@channel.us
def who(request):
    yield {
        "message": {
            "res": "我"
        }
    }
    time.sleep(1)
    yield {
        "message": {
            "res": "是"
        }
    }
    time.sleep(1)
    yield {
        "message": {
            "res": "你叠"
        }
    }

File: apps/js/example_UnaryToStream.js
import { client } from "../../core/client/client.js";


export const rule = {
  who: {
    reg: "^#你是谁",
    priority: 800,
    describe: "你是谁",
  },
};

export async function who(e) {
  let call = client.UnaryToStream({
    file: "example_UnaryToStream",
    function: "who",
  });

  call.on("data", function(response) {
    e.reply(response.message.res);
  });
}